server/plugin/api.go

```diff
@@ -72,11 +72,12 @@ func (p *MatterpollPlugin) InitAPI() *mux.Router {
 	r := mux.NewRouter()
 	r.HandleFunc("/", p.handleInfo).Methods(http.MethodGet)
 	r.HandleFunc("/"+iconFilename, p.handleLogo).Methods(http.MethodGet)
+	// INFO we need to it here, because we would have to put an exception to checkAuthenticity otherwiese
```

Comment:
I tested in Mattermost v6, and I found that the request from message attachment to server seems to send with auth info. So I think this endpoint can/should be placed under `/api/v1`.